.env

```
@@ -1,4 +1,4 @@
-STRIPE_API_KEY=sk_test_51SyF74H6Zvt2fDNN1Sb6LIe5i8ne8U80Lp4LvmDBYNrOWfQ4ffl76q5LrqBNj3XYROKNkj95mFL8HfQn5P1kGfbm00nGV34GAJ
-STRIPE_WEBHOOK_SECRET=whsec_4b5545a57965d662cd8c6d877dcdcbbe6065afedb8d70670f2558a0b60037b72
+STRIPE_API_KEY=sk_test_51SNm1M2Uk0012hZxzfcr36tIrrZ1keuqvAugYsJQ78ObN8WSwXS2TWsYsZkthblx2H7sZFMlngC7RQ1nQDSsiQpe00wtNKHT1g
+STRIPE_WEBHOOK_SECRET=whsec_66a61c2c8954e34629b1c9bb32a4fc43ddce0c11d80ab33d4b2f8a52a6405b64
 MAILTRAP_USERNAME=b085289b2e31b3
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MAILTRAP_USERNAME key should go before the STRIPE_API_KEY key

(UnorderedKey)